crt_portal/cts_forms/migrations/0100_create_reportattachments_20210209_1609.py

```diff
@@ -0,0 +1,57 @@
+        ),
+        migrations.AlterField(
+            model_name='protectedclass',
+            name='value',
+            field=models.CharField(blank=True, choices=[('age', 'Age'), ('disability', 'Disability (including temporary or recovered and including HIV and drug addiction)'), ('family_status', 'Family, marital, or parental status'), ('gender', 'Gender identity (including gender stereotypes)'), ('genetic', 'Genetic information (including family medical history)'), ('immigration', 'Immigration/citizenship status (choosing this will not share your status)'), ('language', 'Language'), ('national_origin', 'National origin (including ancestry and ethnicity)'), ('pregnancy', 'Pregnancy'), ('race/color', 'Race/color'), ('religion', 'Religion'), ('sex', 'Sex'), ('orientation', 'Sexual orientation'), ('none', 'None of these apply to me'), ('other', 'Other reason')], max_length=100, unique=True),
+        ),
+        migrations.AlterField(
+            model_name='report',
+            name='crt_reciept_year',
+            field=models.PositiveIntegerField(blank=True, null=True, verbose_name=django.core.validators.MaxValueValidator(2021)),
+        ),
+        migrations.AlterField(
+            model_name='report',
+            name='intake_format',
+            field=models.CharField(choices=[('web', 'Web'), ('letter', 'Letter'), ('phone', 'Phone'), ('fax', 'Fax'), ('email', 'Email')], default=None, max_length=100, null=True),
+        ),
+        migrations.AlterField(
+            model_name='report',
+            name='last_incident_year',
+            field=models.PositiveIntegerField(blank=True, null=True, verbose_name=django.core.validators.MaxValueValidator(2021, message='Date can not be in the future.')),
+        ),
+        migrations.AlterField(
+            model_name='report',
+            name='primary_complaint',
+            field=models.CharField(choices=[('voting', 'Voting rights or ability to vote affected'), ('workplace', 'Workplace discrimination or other employment-related problem'), ('housing', 'Housing discrimination or harassment'), ('education', 'Discrimination at a school, educational program or service, or related to receiving education'), ('police', 'Mistreated by police, correctional staff, or inmates'), ('commercial_or_public', 'Discriminated against in a commercial location or public place'), ('something_else', 'Something else happened')], default='', max_length=100),
+        ),
```

Comment:
These are unrelated to this card but showed up when running `makemigrations`

Reply:
Thanks for catching these